### PR TITLE
DASH-2624 [Frontend] Rename Labels to Groups

### PR DESCRIPTION
**Pull request by sinisa-nogic**
https://webcomputing.jira.com/browse/DASH-2624

### CODE REVIEW

File: src/appsco-directory-page.html [hunk 1]
```diff
@@ -611,18 +611,18 @@
...
-            addLabel: function(label) {
-                this.$.appscoCompanyLabels.addLabel(label);
+            addGroup: function(group) {
+                this.$.appscoCompanyGroups.addGroup(group);
             },
 
-            removeLabel: function(label) {
-                this.$.appscoCompanyLabels.removeLabel(label);
-            },
+            /*removeLabel: function(group) {
```
Comment (dexterhovis) [inline src/appsco-directory-page.html]: Why comment?

File: src/components/account/appsco-accounts.html [hunk 2]
```diff
@@ -544,7 +544,7 @@
                 return selectedAccounts;
             },
 
-            getSelectedRolesWithLabels: function(labels) {
+            /*getSelectedRolesWithLabels: function(labels) {
```
Comment (dexterhovis) [inline src/components/account/appsco-accounts.html]: Why commented?

File: src/components/account/appsco-contacts.html [hunk 3]
```diff
@@ -550,7 +550,7 @@
                 return false;
             },
 
-            getSelectedContactsWithLabels: function(labels) {
+            /*getSelectedContactsWithLabels: function(labels) {
```
Comment (dexterhovis) [inline src/components/account/appsco-contacts.html]: Why comment?

File: src/components/group/appsco-company-group-info-page.html [hunk 4]
```diff
@@ -101,68 +101,68 @@
...
 
-            setLabel: function(label) {
-                this.label = label;
+            setGroup: function(group) {
+                this.group = group;
             },
 
-            searchLabels: function(term) {
-                this.$.appscoCompanyLabels.search(term);
+            /*searchLabels: function(term) {
```
Comment (dexterhovis) [inline src/components/group/appsco-company-group-info-page.html]: Why commented out?

File: src/components/group/appsco-company-manage-groups-page.html [hunk 5]
```diff
@@ -117,85 +117,85 @@
...
-            _onLabelInfo: function(event) {
-                this._showCompanyLabelInfoPage();
-                this.$.appscoCompanyLabelInfoPage.setLabel(event.detail.label);
+            _onGroupInfo: function(event) {
+                this._showCompanyGroupInfoPage();
+                this.$.appscoCompanyGroupInfoPage.setGroup(event.detail.group);
             },
 
-            _onRemoveLabel: function(event) {
+            /*_onRemoveLabel: function(event) {
```
Comment (dexterhovis) [inline src/components/group/appsco-company-manage-groups-page.html]: Why commented out?

**Comment by sinisa-nogic** [comment]
All commented out method are unused.

**Pull request merged**